src/component/Navbar.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { MdNetworkCell } from "react-icons/md";
+import { BiWifi2 } from "react-icons/bi";
+import { FaBatteryFull } from "react-icons/fa6";
+import moment from "moment";
+
+const Navbar = ({ currentTimeAndDate }) => {
```

Comment:
any reason to pass currentTimeAndDate as a prop, because your using moment to show it  {moment(currentTimeAndDate).format("HH:mm")}
if you put
 {moment().format("HH:mm")} it will result the same, then why the passing? @amitark7